Makefile

```diff
@@ -17,6 +17,7 @@ CLI_PKG     = $(PROJECT_PKG)/cmd/oras
 GIT_COMMIT  = $(shell git rev-parse HEAD)
 GIT_TAG     = $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
 GIT_DIRTY   = $(shell test -n "`git status --porcelain`" && echo "dirty" || echo "clean")
+GO=go1.20
```

Comment:
`go1.20` is a bit confusing that most developers do not have commands like `go1.20` or `go1.19` in their `$PATH`. By following the golang official installation [instruction](https://go.dev/doc/install), most developers end up with just the `go` command.

Reply:
Done

Reply:
Done